app/main.py

```
@@ -595,6 +594,33 @@ async def chat_image(
             pass
     return chat_image
 
+@app.get("/chat/{chat_id}/image/{image_id}")
+async def chat_image(
```

Ruff (0.13.1)

Redefinition of unused `chat_image` from line 580

(F811)